.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```

Comment:
Please do not commit your .vscode config files.

Reply:
Thanks, removed.